Storage/Storage/Model/DashboardCard.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// Represents a card on the Dashboard screen.
+public struct DashboardCard: Codable, Hashable, Equatable {
+    /// The type of dashboard card.
+    public let type: CardType
+
+    /// Whether the card is enabled in the Analytics Hub.
+    public var enabled: Bool
```

Comment:
❓  I am trying to understand how we plan to use the `enabled` boolean. If we plan to use it only for storing in settings, can we just store the enabled card types and assume that the remaining card types are disabled?

Reply:
I'm following the logic implemented in the Analytics Hub, where the unavailable cards are still displayed on the Customize screen but with an Unavailable badge. More info can be found in #12456.